### ISSUE TITLE
init/import: Add a way to specify the SRS

### ISSUE THREAD

**Issue by craigds** [koordinates/sno#86]
In #83 we're adding imports from OGR datasources, starting with SHP.

We can't enable TAB just yet because it doesn't preserve EPSG codes, which makes portability tricky when you want to checkout your repo to GPKG.

If we can allow the user to specify the SRS, we can enable TAB imports.

( Alternatively, find a way to auto-detect them, but it sounds pretty hopeless: https://github.com/OSGeo/gdal/issues/1183#issuecomment-452316269 )

**Comment by rcoup** [comment]
Yeah, I think an optional argument that we make required for TAB/CSV/whatever would be fine.

**Comment by tcwilkinson** [comment]
+1 Yes! This would be very helpful...

Would like to be able to CSV files with WKT geometry or point coordinate fields (X and Y or lat/lng) into kart (and obvious specify the CRS and fields). Would be useful to specify CRS either as epsg/esri code for standard CRS or WKT, OR else import WKT via side car file.

```
kart import places.csv --crs_file=places.wkt --geometry_field="geometry"
```
OR
```
kart import places.csv --crs="EPSG:4326" --coordinate_y="Latitude" --coordinate_x="Longitude"
```
OR
```
kart import places.csv --crs="EPSG:32636" --coordinate_x="Easting" --coodinate_y="Northing"
```

I can see it's been a while though since this was commented. Is it still the plan to include CSV (and things like GeoJSON) import into kart?
